scripts.js

```diff
@@ -187,36 +191,61 @@
   }
 
   function setNameToSay () {
+    totalSyllables = syllable(name)
     if (!hasSpeechSupport) {
       return
     }
     nameToSay = new window.SpeechSynthesisUtterance(name)
+
     nameToSay.pitch = 0.8
     nameToSay.rate = 1.25
     nameToSay.lang = 'en-US'
 
+    var voices = window.speechSynthesis.getVoices()
```

Comment:
Trying to set a deeper voice is a good idea but when I try this locally I don't get a deep voice, I imagine the voices available may be different per device?

For example my phone has a deep voice whereas my computer has a higher voice.

If you could figure out a way to have this consistent would welcome a pull request for this.